model/project_aliases.go

```diff
@@ -221,24 +222,24 @@ func mergeAliases(parserProject *ParserProject, databaseAliases []ProjectAlias)
 // FindAliasInProjectOrRepo finds all aliases with a given name for a project.
 // If the project has no aliases, the repo is checked for aliases.
 func FindAliasInProjectOrRepo(projectID, alias string) ([]ProjectAlias, error) {
-	aliases, err := findAliasInProject(projectID, alias)
+	aliases, shouldExit, err := findMatchingAliasForParserProject(projectID, alias)
 	if err != nil {
-		return aliases, errors.Wrapf(err, "error finding aliases for project '%s'", projectID)
+		return aliases, errors.Wrapf(err, "error checking parser project for project '%s' for aliases", projectID)
 	}
-	if len(aliases) > 0 {
+	if shouldExit {
```

Comment:
shouldExist could be true while aliases is empty. Should we not be fetching it from somewhere else in that case?

Reply:
Do you mean shouldExit? ShouldExit is true when aliases is empty in the case I described earlier, where other patch aliases _are_ defined.

Reply:
If I'm misunderstanding, can you provide an example of what you mean?

Reply:
Ah, I misread it